_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1791,7 +1791,7 @@ mfxStatus VAAPIEncoder::Execute(Task const & task, mfxHDLPair pair)
                     mfxU32 length = (header.bit_length + 7) / 8;
 
                     assert(mfxU32(bsDataStart + m_width*m_height - bsDataEnd) > length);
```

Comment:
no need in this assert, since it is already included in check on line 1795

Reply:
You are right, it is duplicate assertion. I will remove it